transform/models/marts/geo_reference/_geo_reference__models.yml

```diff
@@ -0,0 +1,88 @@
+      - name: geo_id
+        description: >
+          Census block identifier; a concatenation of 2020 Census state FIPS code, 2020
+          Census county FIPS code, 2020 Census tract code, and 2020 Census block number
+      - name: name
+        description: >
+          2020 Census tabulation block name; a concatenation of 'Block'
+          and the tabulation block number
+      - name: maf_tiger_feature_class_code
+        description: MAF/TIGER Feature Class Code
+      - name: maf_tiger_feature_class
+        description: >
+          The lowest-order census defined statistical area. It is an area,
+          such as a city block, bounded primarily by physical features but
+          sometimes by invisible city or property boundaries. A tabulation
+          block boundary does not cross the boundary of any other geographic
+          area for which the Census Bureau tabulates data. The subtypes of
+          this feature are Count Question Resolution (CQR), current, and
+          tabulation census.
+      - name: urban_rural_area
+        description: >
+          Reserved for 2020 Census urban/rural indicator
+          (2020 Urban Areas are not yet defined)
+      - name: housing_unit_count
+        description: 2020 Census housing unit count
+      - name: pop_count
+        description: 2020 Census population count
+      - name: geometry
+        description: The spatial component of geographic features
+  - name: dim_geo_reference_footprints_places__joined
+    description: |
+      This data table is a join of the TIGER shapefile
+      , Places, with the Microsoft Building Footprints
+      data for the state of CA.
+    columns:
+      - name: release
+        description: The version of the data
+      - name: capture_dates_range
+        description: >
+          Each building footprint has a capture date tag associated from 2019-2020
+      - name: place_fp
+        description: Current place FIPS code
+      - name: place_ns
+        description: Current place GNIS code
+      - name: geo_id
+        description: >
+          Place identifier; a concatenation of the current state
+          FIPS code and place FIPS code
+      - name: name
+        description: >
+          Current name and the translated legal/statistical
+          area description for place
+      - name: class_fp_code
+        description: Current FIPS class code
+      - name: class_fp
+        description: Current FIPS class definition
+      - name: maf_tiger_feature_class_code
+        description: MAF/TIGER Feature Class Code
+      - name: maf_tiger_feature_class
+        description: >
+          A legal entity incorporated under state law to provide general-purpose
+          governmental services to a concentration of population. Incorporated
+          places are generally designated as a city, borough, municipality, town,
+          village, or, in a few instances, have a different legal description.
+          - and -
```

Comment:
What's the intended formatting here? Remember that these descriptions are markdown strings.

Reply:
I meant the dashes around the `and`. Is the intent to emphasize? You have markdown at your disposal, so it could be `*and*`.

Reply:
Oh lol that was because it's two meanings. I guess two sentences is fine and I don't need to structure this way.